services/crypto/key/controls.yaml

```
@@ -7,73 +7,124 @@ shared-controls:
       - CCC.C04 # Log all access and changes
       - CCC.C05 # Prevent access from untrusted entities
       - CCC.C06 # Prevent deployment in restricted regions
-      - CCC.C10 # Prevent Data Replication to Destinations Outside of Defined Trust Perimeter
+      - CCC.C10 # Prevent Data Replication to Destinations Outside Perimeter
 
 control-families:
   - name: Logging and Monitoring
-    description: 'TODO: Describe this control family'
+    description: Controls that collect, alert, and retain key-management events.
+
     controls:
       - id: CCC.KeyMgmt.C01
-        title: Alert when key versions are deleted or disabled
-        objective: |
-          Ensure that alerts are generated when key versions are scheduled for deletion or disabled.
+        title: Alert on key-version changes
+        objective: >
+          Generate near-real-time alerts when a KMS key version is disabled or
+          scheduled for deletion, enabling rapid investigation and recovery.
         requirements:
           - id: CCC.KeyMgmt.C01.TR01
             text: |
-              Verify that alerts are triggered when the kms key version is scheduled for deletion.
+              When a key version is scheduled for deletion or disabled, an
+              alert MUST be generated within five minutes.
             applicability:
               - tlp-green
```

```suggestion
              - tlp-amber
              - tlp-red
```

Do we need this for all sensitivity levels? I'd expect if we only need it for some, it would be the higher levels.